Kconfig

```diff
@@ -46,6 +46,10 @@ config HW_LLI
 	bool
 	default n
 
+config DMA_AGGREGATED_IRQ
+	bool
+	default n
+
```

Comment:
I know we don't have *help* for other hidden configs, but it would be nice to have one like here, so we all understand what it does: https://github.com/torvalds/linux/blob/master/arch/Kconfig#L131
Please start adding them if that's not a problem

Reply:
Good suggestion, done.